package.json

```diff
@@ -38,7 +38,7 @@
     "body-parser": "1.18.3",
     "dateformat": "3.0.3",
     "express": "~4.16.4",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/DeepshikhaNEC/iotagent-node-lib.git#issue/372",
```

Comment:
Looking to sibling PR https://github.com/telefonicaid/iotagent-json/pull/458, it uses:

```
"iotagent-node-lib": "git://github.com/Madhu-NEC/iotagent-node-lib.git#selectively_ignore_some_measure",
```

Do they correspond to the same iotagent-node-lib code?

(Anyway, don't worry about this... first PR https://github.com/telefonicaid/iotagent-node-lib/pull/835 will be merged, then this packages.json restored to master)